schemas/HPXMLBaseElements.xsd

```diff
@@ -4608,14 +4613,8 @@
 			<xs:element minOccurs="0" name="StormWindow">
 				<xs:complexType>
 					<xs:sequence>
-						<xs:group ref="SystemInfo"/>
-						<xs:element minOccurs="0" name="GlazingMaterial" type="GlazingMaterial"/>
-						<xs:element minOccurs="0" name="GlassType" type="GlassType"/>
-						<xs:element minOccurs="0" name="FrameType" type="WindowFrameType"/>
-						<xs:element minOccurs="0" name="Location" type="StormLocation"/>
+						<xs:group ref="StormInfo"/>
 						<xs:element minOccurs="0" name="Operable" type="HPXMLBoolean"/>
```

Comment:
@yzhou601 This will result in a breaking change because the order of elements changes. It's probably fine to use the same elements for StormDoor and StormWindow, even if it's a little weird for `Operable`.